### ISSUE TITLE
Reset command example

### ISSUE THREAD

**Issue by gmhhope** [shuzhao-li-lab/PythonCentricPipelineForMetabolomics#68]
## Reset command

Hi Josh, 

I need to have an example of the reset command as the initial attempt fails and it complains the reset in not a valid subcommand. 

My pcpfm version is: '1.0.9'.

```
(base) MLG-JGM357:~ gongm$ pcpfm reset -i /Users/gongm/Documents/projects/Adriana_U01_RZV_imm_persist/git-repo/output/pfmRun0508/HILICpos/
reset is not a valid subcommand
valid commands include:
	 QAQC
	 asari
	 assemble
	 assemble_study
	 batch_correct
	 blank_masking
	 build_empCpds
	 convert
	 delete
	 download_extras
	 drop_missing_features
	 drop_outliers
	 drop_samples
	 finish
	 generate_output
	 impute
	 l1a_annotate
	 l1b_annotate
	 l2_annotate
	 l4_annotate
	 log_transform
	 map_ms2
	 normalize
	 preprocess
	 process_params
	 report
	 summarize
```

Thanks for the help in advance!

Best regards,
Minghao

**Comment by gmhhope** [comment]
Upgrading to '1.0.14' still not working

**Comment by jmmitc06** [comment]
This has been fixed in v1.0.15, the main function was improperly indented in main.
